Sources/FluentMySQLDriver/FluentMySQLConfiguration.swift

```diff
@@ -68,7 +68,7 @@ extension DatabaseConfigurationFactory {
         hostname: String,
         port: Int = 3306,
         username: String,
-        password: String,
+        password: String? = nil,
```

Comment:
```suggestion
        password: String?,
```
Again, don't default to `nil`